src/profile/public/index.html

```diff
@@ -53,111 +53,11 @@
 
 						<ul class="ml-auto navbar-nav">
 
-							<!-- Registration Link -->
-							<li class="nav-item">
-								<a class="nav-link" data-dismiss="dropdown" data-target="#register-form" data-toggle="dropdown" href="#" id="register">Register</a>
-							</li>
-
```

Comment:
We no longer need registration, because the UserLib is generating our identity for us.